docs/self-managed/operational-guides/update-guide/870-to-880.md

```diff
@@ -254,3 +254,25 @@ In that case, we append the following records:
 - `ASSIGN`
 - `ASSIGNING`
 - `ASSIGNED`
+
+### Zeebe record types
+
+With the introduction of the Camunda Exporter, the Elasticsearch and OpenSearch Exporter no longer exports all record types by default.
+Meaning that not all `zeebe-record` indices will be populated.
+
+::note This only affects records exported with the 8.8 version. During the migration, left over 8.7 records are exported as normal.
+::
+
+The record types that continue to be exported by default are the following:
+
+- `DEPLOYMENT`
+- `PROCESS`
+- `PROCESS_INSTANCE`
+- `VARIABLE`
+- `USER_TASK`
+- `VARIABLE`
```

Comment:
You have `VARIABLE` listed twice. I think this should be `INCIDENT`? Optimize needs to import incidents, so it should continue to be in the default set, regardless